docs/index.rst

```diff
@@ -1,16 +1,17 @@
-Jupyter Client |version|
-========================
+Jupyter Protocol |version|
+==========================
 
-This package provides the Python API for starting, managing and communicating
-with Jupyter kernels.
+This package provides the Python API for message composition relative to Jupyter
+kernels. For information related to the launch, communication and lifecycle
+management of Jupyter kernels, please refer to the `Jupyter Kernel Management
+documentation <https://jupyter-kernel-mgmt.readthedocs.io/en/latest/index.html>`_.
 
 .. important::
    This document contains the authoritative description of the
    Jupyter messaging protocol. All developers are strongly encouraged to
```

Comment:
Let's comment this out or something until the focus has moved from jupyter_client. Maybe even add a note referring to the equivalent document in jupyter_client for now.